packages/x-components/src/components/panels/base-tabs-panel.vue

```diff
@@ -0,0 +1,331 @@
+<template>
+  <section v-if="getTabs().length > 0" class="x-tabs-panel" data-test="base-tabs-panel">
+    <component
+      :is="animation"
```

Comment:
I'm wondering wether we should apply an animation to the content instead of the tabs list. In the end I think the part which will change the most with users interactions will be that content, isn't it? Each time the user clicks a different tab.

Reply:
I was not completely sure about this, but I thought about adding the animation to the list (to animate how the list is presented, or new tabs could appear) and not to the tab itself as this last can be added anyway through the `tab` slot.  This way both animations can be added in one way or another.

Reply:
I've added a new prop to have an animation for the tab content and left another animation for the tabs list.